internal/serve/httphandler/stellar_toml_handler.go

```diff
@@ -40,15 +40,17 @@ func (s *StellarTomlHandler) horizonURL() string {
 
 // buildGeneralInformation will create the general informations based on the env vars injected into the handler.
 func (s *StellarTomlHandler) buildGeneralInformation(ctx context.Context, req *http.Request) string {
-	distributionPublicKey, err := s.DistributionAccountResolver.DistributionAccountFromContext(ctx)
-	if err != nil {
+	accounts := "["
+	if perTenantDistributionAccount, err := s.DistributionAccountResolver.DistributionAccountFromContext(ctx); err != nil {
 		log.Ctx(ctx).Warnf("Couldn't get distribution account from context in %s%s", req.Host, req.URL.Path)
-		distributionPublicKey = s.DistributionAccountResolver.HostDistributionAccount()
+		accounts += "\"" + s.DistributionAccountResolver.HostDistributionAccount() + "\"" // fallback to host distribution account
```

Comment:
❓Do we actually need this fallback ? Would receiver registration break if we don't put a distribution account here ?